.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,86 @@
+name: "Bug report 🐛"
+description: Report errors or unexpected behavior
+title: ''
+labels: 
+- Issue-Bug
+- Triage-Needed
+assignees: ''
+issue_body: true
+body:
+- type: markdown
+  attributes:
+    value: |
+      Please make sure to [search for existing issues](https://github.com/microsoft/PowerTOys/issues) before filing a new one!
+
+- type: input
+  attributes:
+    label: Microsoft PowerToys version
+    placeholder: "0.33.1"
+    description: |
+      Hover over system tray icon or look at Settings
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Running as admin
+    options:
+      - 'Yes'
+      - 'No'
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Utility with issue?
+    multiple: true
+    options:
+      - General
+      - ColorPicker
+      - FancyZone
+      - Image Resizer
+      - Keyboard manager
+      - MD Preview
+      - PowerRename
+      - PowerToys Run
+      - Shortcut Guide
+      - SVG Preview
+      - SVG Thumbnail
+      - Installer
+      - System tray interaction
```

Comment:
```suggestion
    options:
      - General
      - ColorPicker
      - FancyZone
      - Image Resizer
      - Keyboard manager
      - MD Preview
      - PowerRename
      - PowerToys Run
      - Shortcut Guide
      - SVG Preview
      - SVG Thumbnail
      - Installer
      - System tray interaction
      - Settings window
```

Reply:
@htcfreek 
can you edit this suggestion to only suggest the actual change?